Delfino/DualCore/SmartWinchDualCore/kinematics.c

```diff
@@ -183,8 +133,7 @@ float min_max_diff(float g, float h, float i, float j) {
     float min = 1000000;
     float max = 0;
     float arraytest [] = {g,h,i,j};
-    int m;
-    for (m = 0;m<4;m++) {
+    for (int m = 0;m<4;m++) {
```

Comment:
dont use for (int m = 0;m<4;m++)

use this instead:

int m;
for (m = 0;m<4;m++)

the compiler doesnt support var declaration inside for parameter